UnityProjects/MRTKDevTemplate/Packages/manifest.json

```diff
@@ -31,9 +31,8 @@
     "com.unity.project-auditor": "https://github.com/Unity-Technologies/ProjectAuditor.git",
     "com.unity.test-framework": "1.1.33",
     "com.unity.textmeshpro": "3.0.6",
-    "com.unity.timeline": "1.4.8",
+    "com.unity.timeline": "1.6.4",
     "com.unity.ugui": "1.0.0",
-    "com.unity.xr.arcore": "4.1.12",
```

Comment:
Intentional removal of the ARCore package?

Reply:
Oops, I don't think so.

Reply:
Fixed, and bumped to 5.0.3